src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -184,6 +184,8 @@ private void SetGlobalErrorIfTrue(bool arg)
             }
         }
 
+        // Returns the MethodSymbol for the assembly entrypoint.  If the user has a Task returning main,
+        // this function returns the synthesized Main MethodSymbol.
```

Comment:
Do we need to specify the "MoveNext" as an entry point for debug purposes? 
I think it was required for some debugging scenarios like F11 to work if there is any code that runs before the actual Main, but I am not sure if that applies here.